src/index.ts

```diff
@@ -0,0 +1,15 @@
+import { request } from "@octokit/request";
+import getUserAgent from "universal-user-agent";
+
+import { version } from "../package.json";
```

Comment:
I think that won’t work. Create `src/version.ts` instead, compare

- https://github.com/octokit/request.js/blob/d09b100f5204d32b6f369c203de2e66f015d781e/src/version.ts
- https://github.com/octokit/request.js/blob/d09b100f5204d32b6f369c203de2e66f015d781e/package.json#L98-L115

This will replace the version after building, before publishing to npm